FormulaAbstract/tezos.rb

```diff
@@ -13,9 +13,9 @@ class << self
     end
     homepage "https://gitlab.com/tezos/tezos"
 
-    url "https://gitlab.com/tezos/tezos.git", :tag => "v8.1", :shallow => false
+    url "https://gitlab.com/tezos/tezos.git", :tag => "v8.2", :shallow => false
 
-    version "v8.1-1"
+    version "v8.2-1"
```

Comment:
Same reason as in native packaging. Something unrelated to the Tezos sources can change, e.g. we can add background services in the scope of #130

Reply:
"-N" suffix indicates "release number" if we borrow RPM terminology - if a new build is necessary because of changes/fixes in packaging itself (e.g. package metadata needs an update - description, maintainer email etc, or a required file wasn't included)

Reply:
:ok_hand:  Got it, makes sense to me.

I was thinking of adding the "release number" only with the first non-version update, but this is a lot more consistent.